client/my-sites/stats/stats-interval-dropdown/style.scss

```diff
@@ -1,18 +1,42 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@automattic/components/src/styles/typography.scss";
 
+.stats-interval-dropdown,
 .stats-interval-dropdown__container {
 
-	width: 196px;
+	font-style: normal;
+	--gray-gray-10: #c3c4c7;
```

Comment:
Do you think it'll a bit more clearer if we define/override these at the very beginning of the CSS/file?